src/common/netif.cpp

```diff
@@ -31,6 +31,30 @@
 
 namespace {
 
+//! Return CNetAddr for the specified OS-level network address. If a length is provided, it is checked for validity.
```

Comment:
```
If a length is provided, it is checked for validity.
```
Maybe it would be better to mention that it is checked for validity inside of `SetSockAddr()`, or is this detail that the caller of `FromSockAddr()` does not need to know?

Reply:
Right, the logic here has been swapped since this comment was written 😄 

i think that sentence is redundant. That the length is checked is implied, why pass it otherwise? The line below already describes which length is used when it's not passed.